README.md

```diff
@@ -54,7 +54,7 @@ This sample is also provided in Try .NET. Currently using this in Try .NET is av
 
 To get to the try .MD file that walk over the sample click here:
```

Comment:
1. There's a lot of capitalization inconsistency in this blurp. Can we keep it as all "Try .NET" throughout?
2. "that walk over the sample" should be "that walks over the sample"